commodore/component-template/{{ cookiecutter.slug }}/.gitignore

```diff
@@ -1,7 +1,18 @@
-_archive/
-_public/
+# Commodore
 .cache/
 helmcharts/
 manifests/
 vendor/
 jsonnetfile.lock.json
+crds/
+compiled/
+
+# Editors
```

Comment:
This is subject to grow out of hand. We do not have an agreement on which editor to use. In consequence, every editor can be used. Adding ignores for every editor is nothing I would be willing to do.

As a git user, I consider it to be best practice, to have a global git ignore to exclude things specific to my system. This include also such things like vim swap files, the all famous and anoying `.DS_Store` macOS etc.

Reply:
I agree that editor-specific things don't belong in repo .gitignores but rather in a global .gitignore or (`.git/info/exclude`). However, the contents of this gitignore are verbatim from https://github.com/projectsyn/modulesync-control/blob/master/moduleroot/.gitignore.erb, so if we want to change it here, we need to change it there too.

Reply:
Updated in latest version and removed from modulesync-control: https://github.com/projectsyn/modulesync-control/commit/668a2b3edfef9e129100a8aecda1dd08f9a891bf